specification/app/resource-manager/Microsoft.App/stable/2022-05-01/ContainerApps.json

```
@@ -752,6 +762,46 @@
         }
       }
     },
+    "IpRestrictionRules": {
+      "description": "incoming ip addresses restriction rules",
+      "type": "object",
+      "required": [
+        "name",
+        "ipAddress",
+        "action"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of ip restriction rule",
+          "type": "string"
+        },
+        "ipAddress": {
+          "description": "ipAddress is the CIDR to match incoming traffic against",
+          "type": "string"
+        },
+        "action": {
+          "description": "ALLOW or DENY set ip address's",
+          "enum": [
+            "ALLOW",
```

nit: Can these be Pascal case?

```suggestion
            "Allow",
```

---

done